src/Components/Shifting/models.ts

```diff
@@ -39,6 +39,6 @@ export interface IShift {
   is_assigned_to_user: boolean;
   created_by: number;
   last_edited_by: number;
-  patient: string | PatientModel;
+  patient: string | PatientModel | number;
```

Comment:
Same here, why being typed as number?
```suggestion
  patient: string | PatientModel;
```